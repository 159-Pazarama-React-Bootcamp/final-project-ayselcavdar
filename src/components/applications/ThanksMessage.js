import styles from './Applications.module.css';

const ThanksMessage = () => {
  return (
    <div className={styles['message-container']}>
      <p>Teşekkürler! Başvurunuz başarıyla alınmıştır.</p>
      <p>
        Başvuru durumunuzu aşağıda size verilen başvuru kodu ile sorgulayabilir
        veya detaya tıklayarak öğrenebilirsiniz.
      </p>
    </div>
  );
};

export default ThanksMessage;
