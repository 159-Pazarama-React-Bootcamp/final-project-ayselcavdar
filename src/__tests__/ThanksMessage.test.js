import render from 'react-test-renderer';
import ThanksMessage from '../components/applications/ThanksMessage';

describe('<ThanksMessage /> rendering', () => {
  it('first p tag text should be given title prop value', () => {
    const wrapper = shallow(
      <ThanksMessage title={'test'}/>
    );
    expect(wrapper.find('p').first().text()).toEqual('test');
  });

  it('ThanksMessage snapshot test', () => {
    const comp = render.create(<ThanksMessage title="Hello" />);
    let tree = comp.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
