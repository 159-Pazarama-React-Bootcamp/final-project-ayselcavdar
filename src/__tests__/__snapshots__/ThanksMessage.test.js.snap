// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThanksMessage /> rendering ThanksMessage component snapshot test 1`] = `
<div
  className="message-container"
>
  <p>
    Hello
  </p>
  <p />
</div>
`;

exports[`<ThanksMessage /> rendering ThanksMessage snapshot test 1`] = `
<div
  className="message-container"
>
  <p>
    Hello
  </p>
  <p />
</div>
`;

exports[`ThanksMessage component snapshot test 1`] = `
<div
  className="message-container"
>
  <p>
    Hello
  </p>
  <p />
</div>
`;

exports[`ThanksMessage render check 1`] = `
<div
  className="message-container"
>
  <p>
    Hello
  </p>
  <p />
</div>
`;
